client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -257,16 +273,20 @@ void main(void)
 #endif
 		perspective_factor = pFactor;
 
-		if (f_timeofday < 0.2) {
+		sunTint = vec3(1.0);
+		if (f_timeofday < 0.21) {
```

Comment:
What's the reason for these magic number changes?

Reply:
Pretty much to make the shadow intensity actually correspond to sunrise/sunset correctly. The original values were just slightly off. Also the transitions between shaded/unshaded were not equally long in the original code. This wasn't very notable before, but with a lot of the effects from this and the Vol. 1 PR it can be.

Reply:
I was about to ask the same thing in `object_shader/opengl_vertex.glsl`.  Are they calculated, or found by try&error? I would very much appreciate it if you could add a short in-line comment in both shaders for reference.